.gitignore

```diff
@@ -1,3 +1,4 @@
 node_modules
 .idea
 npm-debug.log
+build
```

Comment:
this might cause travis not to push the built files to gh-pages, unless we dynamically remove this during the build step